File: task_1.c
#include <stdio.h>
#include <string.h>

void check_num(int n);
void check_name(const char*  s);
void print_multiples_of_three(int array[], int size);

int main(void)
{
    // the firts part of task_1
    int num; 
    printf("Enter a number: ");
    scanf("%d", &num); 
    check_num(num); 
    
    // the second part of task_1
    char name[100]; 
    printf("Enter a name: ");
    scanf("%s", name); 
    check_name(name);
     
    // the third part of task_1
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int array[n]; 
    printf("Enter %d integers:\n", n);
    for (int i = 0; i < n; i++) 
    {
        scanf("%d", &array[i]); 
    }
    print_multiples_of_three(array, n);

}   

void check_num(int n)
{
    if (n > 7)
    {
        printf("Hello\n");
    }
}

void check_name(const char* s)
{
    if (strcmp(s, "John") == 0)
    {
        printf("Hello, John\n");
    }
    else
    {
        printf("There is no such name\n");
    }
}

void print_multiples_of_three(int array[], int size) 
{
    for (int j = 0; j < size; j++) 
    {
        if (array[j] % 3 == 0) 
        {
            printf("%d ", array[j]);
        }
    }
    printf("\n");
}

File: assesment/task_1.py
def main():
    num = int(input("Enter a number: "))
    number(num)

    name = input("Enter a name: ")
    greeting(name)

    my_array = []
    size = int(input("What is the length of the array? "))
    for i in range(size):
        elem = int(input(f"Enter element {i + 1}: "))
        my_array.append(elem)
    multiples(my_array)


def number(num):
    if num > 7:
        print("Hello")


def greeting(name):
    if name == 'John':
        print(f"Hello, {name}")
    else:
        print("There is no such name")


def multiples(array):
    for elem in array:
        if elem % 3 == 0:
            print(elem)


main()
